apps/microtvm/reference-vm/base-box-tool.py

```diff
@@ -360,6 +361,9 @@ def do_build_release_test_vm(
     found_box_line = False
     with open(release_test_vagrantfile, "w") as f:
         for line in lines:
+            # Skip setting version
+            if "config.vm.box_version" in line:
```

Comment:
Why ignore the version here?

Reply:
because version only used for released boxes, but we are testing this one locally